chall01/llahti.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+import sys
+
+morse = [".-",
+"-...",
+"-.-.",
+"-..",
+".",
+"..-.",
+"--.",
+"....",
+"..", 
```

Comment:
like this array is not really describeful like could be a Dict, I would recommend put it in a one-line.